exercises/flatten-array/FlattenArrayExample.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+func constructList(elements: Any?...) -> [Any?] {
```

Comment:
I'm not convinced that this `constructList` function makes things more clear. 

`[1,2,3]` vs. `constructList(elements: 1, 2, 3)` ? What do you think?

Or maybe since it is used by the tests and not the implementation we should move it to be a helper function on the test target?